### DISCUSSION TITLE
An error occurred (PermanentRedirect) when calling the GetBucketAcl operation

### DISCUSSION THREAD

**Discussion by axosecurity** [sa7mon/S3Scanner#312] [General]
### Existing Issues

- [X] I've searched the [current issues](https://github.com/sa7mon/S3Scanner/issues)

### Command

s3scanner scan -b flaws.cloud 

### Observed behavior (what happened?)

When i try to scan a bucket 

s3scanner scan -b flaws.cloud 
Bucket scan raised exception: flaws.cloud - An error occurred (PermanentRedirect) when calling the GetBucketAcl operation: The bucket you are attempting to access must be addressed using the specified endpoint. Please send all future requests to this endpoint.

how to solve this 

### Expected behaviour

scan  the buket show me result 

### Debug output

noting

**Comment by sa7mon** [comment]
Hi, please make sure you are using the latest version (v3.0.4 currently). I scanned the bucket just now and it worked fine:

```
$ s3scanner -bucket flaws.cloud
INFO exists    | flaws.cloud | us-west-2 | AuthUsers: [] | AllUsers: [READ]
```
